local/mac/README.md

```diff
@@ -27,7 +27,7 @@ These playbooks will:
      * Shared Folders (create these folders on your mac owned by your user)
          * /docker_shared
               * Note: With version 3.9.0 of oc client, the 'origin' directory should no longer exist.  
-         * persistent volumes
+         * /persistedvolumes
```

Comment:
Again, changed this back to match what we originally had so name is explicit.

Whatever we change this name to we also have to modify the Docker settings on Mac so it refers to the shared volume.   The docker settings are created manually and we are handling them just with documentation at moment.